tests/components/verisure/test_config_flow.py

```diff
@@ -392,9 +370,10 @@ async def test_options_flow_code_format_mismatch(hass: HomeAssistant) -> None:
 
     result = await hass.config_entries.options.async_init(entry.entry_id)
 
-    assert result["type"] == FlowResultType.FORM
-    assert result["step_id"] == "init"
-    assert result["errors"] == {}
+    assert result.get("type") == FlowResultType.FORM
+    assert result.get("step_id") == "init"
+    assert result.get("errors") == {}
+    assert "flow_id" in result
```

Comment:
It could be worth pulling out this common validation from most of the tests into a helper function.

Reply:
They are not as common as they seem, they all slightly vary.

I personally love the explicitness of this, makes the tests easier to read. Helpers don't always contribute to readable code.